engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/error/NumberParseError.java

```diff
@@ -0,0 +1,19 @@
+package org.enso.interpreter.node.expression.builtin.error;
+
+import java.util.List;
+
+import org.enso.interpreter.dsl.BuiltinType;
+import org.enso.interpreter.node.expression.builtin.Builtin;
+import org.enso.interpreter.runtime.callable.atom.Atom;
+
+@BuiltinType
+public final class NumberParseError extends Builtin {
+  @Override
+  protected final List<Cons> getDeclaredConstructors() {
```

Comment:
CCing @jdunkerley - this is the place where I need to change the name of declared constructor and thus I couldn't use `UniquelyConstructibleBuiltin` as it existed in `develop` branch.